rdt/transformers/numerical.py

```diff
@@ -109,8 +112,7 @@ def _learn_rounding_digits(data):
                     return decimal
 
         # Can't round, not equal after MAX_DECIMALS digits of precision
-        warnings.warn(
-            f"No rounding scheme detected for column '{name}'. Data will not be rounded.")
+        LOGGER.info(f"No rounding scheme detected for column '{name}'. Data will not be rounded.")
```

Comment:
one minor comment: I think the use of string literals are preferred for logging over f-strings. https://google.github.io/styleguide/pyguide.html#3101-logging